src/sentry/static/sentry/app/components/modals/inviteMembersModal/index.tsx

```diff
@@ -285,7 +285,7 @@ class InviteMembersModal extends AsyncComponent<Props, State> {
   }
 
   get willInvite() {
-    return this.props.organization.access.includes('member:write');
+    return this.props.organization.access?.includes('member:write');
```

Comment:
I think there's a deeper problem with the organization being "lost", since access should always exist in `organization`. I don't really know what's going on, but my first bet would be `withLatestContext`

Reply:
Yeah, agreed. I was looking at this comment https://github.com/getsentry/sentry/blob/master/src/sentry/static/sentry/app/views/settings/account/accountSettingsLayout.jsx#L30

Reply:
Should have left a comment here I think, we discussed this a bit over hangouts.

It's def a deeper problem. Without access the user is going to be shown the 'send invite requests' modal, which may be confusing if they're an org owner.

Reply:
Hm we could close this and look into replacing `withLatestContext`. How does that sound?

Reply:
Can we keep this fix and also look into `withLatestContext`?
I think having defensive code here and checking to see if access is there or not is a good idea.